README.md

```diff
@@ -1,2 +1,321 @@
-# Quantum Network Explorer CLI
-Command Line Interface to interact with the Quantum Network Explorer
+# Quantum Network Explorer ADK
+Application Development Kit to interact with the Quantum Network Explorer. With the QNE-ADK you can create your application and experiment and run it locally using the commands. 
+
+## Prerequisites
+- A modern Linux 64-bit (x86_64) operating system. If you don’t have linux you could run it via virtualization, e.g. using VirtualBox. If you have Windows 10 you can also use the [Bash on Ubuntu](https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/) subsystem.
+- Python version 3.6 or higher and pip version 19 or higher.
+- NetQASM makes use of SquidASM for which you need credentials in order to use it. These credentials can either be obtained by registering on the forum of [NetSquid](https://forum.netsquid.org/ "NetSquid").
+
+
+## Installation
+To install all the required packages, execute the following commands:
+
+```
+pip install qne-adk
+```
+
+After installing the qne-cli, you can install SquidASM. Replace '{netsquid-user-name}' and '{netsquid-password}' with the credenials you registered on [NetSquid](https://forum.netsquid.org/ "NetSquid"):
+
+```
+pip install squidasm --extra-index-url=https://{netsquid-user-name}:{netsquid-password}@pypi.netsquid.org
+```
+
+Now everything should be setup and ready in order to create your own applications and experiments and run them on the simulator!
+
+
+## Commands
```

Comment:
1. Delete the commands we don't have yet.
2. We have added application delete. 
3. Get the information form the latest version from the repository. Some commands have changed parameters and options (e.g. experiment delete and application delete). Also remote shouldn't be explained.